static/edx-pl/sass/_build.scss

```diff
@@ -0,0 +1,41 @@
+// ------------------------------
```

Comment:
NOTE: edX Pattern Library compile (not used directly in certs, since certs has its own compile that references shared assets).